downloads/downloads-impl/src/main/java/com/duckduckgo/downloads/impl/UrlFileDownloadCallManager.kt

```diff
@@ -63,10 +57,4 @@ class RealUrlFileDownloadCallManager @Inject constructor() : UrlFileDownloadCall
         Timber.d("Adding download $downloadId")
         callsMap[downloadId] = call
     }
-
-    override fun onExit() {
-        // this callback will only be called when the AppTP process is enabled and the user swipe closes the app :shrug:
-        // to be consistent with the behavior when AppTP process is disabled, we want to cancel downloads
-        callsMap.keys.forEach { remove(it) }
-    }
```

Comment:
Removing this for now. This will basically cause downloads to continue when AppTP is enabled and user closes app. I will discuss this in the context of product review